moskito-extensions/moskito-nginx/src/main/java/net/anotheria/moskito/core/stats/impl/RateValueHolder.java

```diff
@@ -0,0 +1,298 @@
+package net.anotheria.moskito.core.stats.impl;
+
+import net.anotheria.moskito.core.stats.Interval;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+/**
+ * Designed to measure values on 'per-second' basis.
+ * Use only with values that are constantly increasing or decreasing but not both.
+ *
+ * @author dzhmud
+ */
+public class RateValueHolder extends AbstractValueHolder {
```

Comment:
Can you explain why we need this class? why wouldn't regular stats do? Also even I saw this class only in github view, I have the feeling it acts very differently to all the other stat classes